src/hdtSkyrimBody.cpp

```
@@ -45,7 +43,8 @@ namespace hdt
 
 	void SkyrimBody::internalUpdate()
 	{
-		if (m_disabled) return;
+		if (m_disabled)
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)